tests/BaldinofRoadRunnerBundleTest.php

```diff
@@ -17,7 +17,7 @@
 
 class BaldinofRoadRunnerBundleTest extends TestCase
 {
-    public function test_it_loads_sentry_middleware_if_needed()
+    public function testItLoadsSentryMiddlewareIfNeeded()
```

Comment:
Can you revert all renamed tests and add `'php_unit_method_casing' => false` in `.php_cs.dist`

Reply:
Done!